dnc/train.py

```diff
@@ -21,8 +21,7 @@
 import tensorflow as tf
 import sonnet as snt
 
-import dnc
```

Comment:
This file is really an example script, not a core part of the package. It should sit outside the dnc subdirectory and import the dnc module, in the same style as external users' training scripts.